ZenPacks/zenoss/Dashboard/browser/resources/js/defaultportlets.js

```diff
@@ -394,6 +396,12 @@
                 store: store,
                 displayField: 'name',
                 valueField: 'uid',
+                locationRegExp: new RegExp('/zport/dmd/Locations'),
```

Comment:
validate combo when it changes it's value. Will show validation error 'The value in this field is invalid'.